src/libs/startup/configStore;.test.ts

```diff
@@ -0,0 +1,34 @@
+import { partial } from 'src/testing/test-utils';
+
+import { AppConfigSettings, loadedConfigStore, resetConfigStore, setLoadedConfigStore } from './configStore';
```

Comment:
Is that a typo in the file name?

Reply:
I don't this so... file exptension are ignosred on import filenames... and it's working without issue in build/test run.